tests/dag_processing/test_manager.py

```diff
@@ -343,14 +344,36 @@ def test_file_paths_in_queue_sorted_random_seeded_by_host(
         manager.prepare_file_path_queue()
         assert manager._file_path_queue == expected_order
 
+    @pytest.fixture
+    def change_platform_timezone(self, monkeypatch):
+        monkeypatch.setenv("TZ", "Europe/Paris")
+
+        # propagate new timezone to C routines (unix only)
```

Comment:
Would be a good to add the reference in the comment to explain why Windows doesn’t need this.

Otherwise looks good to me.